lib/engine/game/base.rb

```diff
@@ -3419,6 +3421,16 @@ def second_icon(corporation); end
       def corp_loans_text
         'Loans'
       end
+
+      def check_player_range!
+        min_players = self.class.min_players(@optional_rules, @players.size)
+        max_players = self.class::PLAYER_RANGE[1]
```

Comment:
There are variants that limit the maximum number of players, so it would make sense to have a call to a `max_players` method here.